site/docs/.vuepress/components/ThankYou.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts" setup>
+import { reactive } from "vue";
+
+const contributor = reactive<Record<string, string>>({
+  name: "",
+  photo: "",
+  href: "",
+});
+
+async function load() {
+  const res = await fetch(
+    "https://raw.githubusercontent.com/grammyjs/grammY/main/.all-contributorsrc"
+  );
+  if (res.status == 200) {
+    const { contributors } = await res.json();
+    const contributor_ =
+      contributors[Math.floor((Date.now() / 86400) % contributors.length)];
```

Comment:
I still wanna find something better here than going through the list chronologically.

Reply:
It doesn’t even work as is.

Reply:
Your bracket placement is weird, too. You're doing mod on float now?

Reply:
It’s Prettier who did that. It also works in Node CLI.

Reply:
I was trying to say that you should have done `(Math.floor(Date.now() / 86400) % contributors.length)` not `Math.floor((Date.now() / 86400) % contributors.length)`

Reply:
But it does not matter anymore

Reply:
Gut. Danken fûr your footercontributorthankyounoterandomizationalgorithm.

Reply:
Ahahah rica dikim

Reply:
*Spas dikim